tests/test_ipython.py

```diff
@@ -141,6 +141,22 @@ def test_artifact_codes(run_cell):
     )
 
 
+def test_lineapy_import_included_in_artifact(run_cell):
```

Comment:
One suggestion is to add a test particularly checking for the case what happens when `import lineapy` is included as well